internal/schema/fabric_common_mapping_helpers.go

```diff
@@ -0,0 +1,230 @@
+package schema
+
+import (
+	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
+	"github.com/equinix/terraform-provider-equinix/internal/converters"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func OrderToFabric(schemaOrder []interface{}) v4.Order {
+	if schemaOrder == nil {
+		return v4.Order{}
+	}
+	order := v4.Order{}
+	for _, o := range schemaOrder {
+		orderMap := o.(map[string]interface{})
+		purchaseOrderNumber := orderMap["purchase_order_number"]
+		billingTier := orderMap["billing_tier"]
+		orderId := orderMap["order_id"]
+		orderNumber := orderMap["order_number"]
+		order = v4.Order{PurchaseOrderNumber: purchaseOrderNumber.(string), BillingTier: billingTier.(string), OrderId: orderId.(string), OrderNumber: orderNumber.(string)}
+	}
+	return order
+}
+
+func OrderToTerra(order *v4.Order) *schema.Set {
```

Comment:
Since you're moving these `fabric_common_` files anyway, I recommend creating a separate directory/package for them, perhaps:
- `internal/fabric/schema/mapping.go`
- `internal/fabric/schema/schemas.go`

Having it in a separate package helps clarify that this `OrderToTerra`, for example, is only useful in the context of Fabric integrations and should probably not be modified by other teams.

Reply:
Yea, good point. Will make that change.